awx/ui_next/src/api/mixins/Notifications.mixin.js

```diff
@@ -59,8 +60,7 @@ const NotificationsMixin = parent =>
      * @param[resourceId] - id of the base resource
      * @param[notificationId] - id of the notification
      * @param[notificationType] - the type of notification, options are "success" and "error"
-     * @param[associationState] - Boolean for associating or disassociating,
-     *                            options are true or false
+     * @param[associationState] - Boolean for associating or disassociating, options are true or false
```

Comment:
Boolean flags in func params can be confusing. It might be worth splitting this into an `associateNotificationTemplate` and `disasociateNotificationTemplate` functions at some point. This would also cut down the number of function arguments.